app/views/recipes/show.html.erb

```diff
@@ -0,0 +1,59 @@
+<div class="recipes-container">
+  <h1><%= @recipe.name %></h1>
+
+  <div class="toggle">
+    <%= button_to @recipe.public? ? 'Make private' : 'Make public', toggle_public_path(id: @recipe.id), method: :post, class: 'toggle_btn', form_class: 'toggle' %>
+  </div>
+
+  <div class="recipes-features">
+  <p class="time"><span>Preparation Time:</span><span><%= @recipe.preparation_time %> hours</span></p>
+  <p class="time"><span>Cooking Time:</span> <span><%= @recipe.cooking_time %> hours</span></p>
+
+  <p class="steps">Steps to make the recipe</p>
+  <% if params[:show_full].present? %>
+    <p class="description">
+      <%= @recipe.description %>
+      <%= link_to 'Show less', recipe_path(show_full: nil), class: 'show_more_btn' %>
+    </p>
+  <% else %>
+    <p class="description">
+      <%=  @recipe.description.split[0...50].join(' ') %>
+      <% if @recipe.description.split.size > 50 %>
+        <%= link_to 'Show more', recipe_path(show_full: true), class: 'show_less_btn' %>
+      <% end %>
+    </p>
+  <% end %>
+  </div>
+
+<div class="back_food_list_btn">
+    <div class="back_food">
+        <%= link_to "Generate shooping list", shoppinglist_index_path(recipe_id: @recipe.id), class:"food_btn" %>
+    </div>
+    <div class="list_btn">
+      <button class="recipe_btn"><%= link_to 'Add ingredient', recipe_foods_path(recipe_id: @recipe.id) %></button>
+      <button type="button" class="recipe_btn"><%= link_to "Go to public recipes", public_recipes_path %></button>
+      <button class="recipe_btn"><%= link_to 'Edit', edit_recipe_path(id: @recipe.id), class: 'edit_btn' %></button>
```

Comment:
- [ ] @team great job implementing the changes requested, however the got to public recipes and edit button are not part to the design kindly remove them from this section, this will make your design match the wireframe layout given.